src/Mod/Fem/App/FemPostPipelinePyImp.cpp

```diff
@@ -50,6 +51,83 @@ PyObject* FemPostPipelinePy::read(PyObject* args)
         PyMem_Free(Name);
         Py_Return;
     }
+
```

Comment:
Use `PyErr_Clear()` here.
`PyArg_ParseTuple` sets an error when it returns `false`.
Python interface wil fail (probably undefined behavior) if continue without handling the error.
https://docs.python.org/3/c-api/exceptions.html#exception-handling

Reply:
ahh that whole code section was just forgotten to be removed... I deleted it entirely